jwst/ami/ami_analyze_step.py

```diff
@@ -11,6 +11,8 @@ class AmiAnalyzeStep(Step):
     spec = """
         oversample = integer(default=3, min=1)  # Oversampling factor
         rotation = float(default=0.0)           # Rotation initial guess [deg]
+        set_psf_offset_find_rotation = string(default='0.0 0.0') # Psf offset values to use to create the model array
```

Comment:
Definitely agree that shorter names would be good for both of these. Perhaps just `psf_offset` and `rotation_search`?